src/rust/sdk/example/src/main.rs

```diff
@@ -0,0 +1,719 @@
+use std::str::FromStr;
+
+use aptos_sdk::{
+    bcs,
+    move_types::{
+        ident_str,
+        language_storage::{ModuleId, StructTag, TypeTag},
+    },
+    rest_client::{aptos_api_types::MoveModuleId, FaucetClient},
+    types::{
+        account_address::AccountAddress, transaction::EntryFunction, LocalAccount, APTOS_COIN_TYPE,
+    },
+};
+use clap::Parser;
+use econia_sdk::{
+    entry::*,
+    errors::EconiaError,
+    types::order::{Restriction, SelfMatchBehavior, Side},
+    view::{EconiaViewClient, PriceLevel},
+    EconiaClient, EconiaResult,
+};
+
+/*
+Running the script:
+
+This script is documented at https://econia.dev/off-chain/rust-sdk.
+
+For a quick start, run the following:
+
+# install and run an aptos node
+brew install aptos # only if necessary
+mkdir aptos && cd aptos
+aptos node run-local-testnet --with-faucet
+
+# export urls in env
+export APTOS_NODE_URL=http://0.0.0.0:8080
+export APTOS_FAUCET_URL=http://0.0.0.0:8081
+
+# get the econia code
+git clone https://github.com/econia-labs/econia.git
+
+# deploy a faucet
+cd ./econia/src/move/faucet
+aptos init --profile econia_faucet_deploy
+export FAUCET_ADDR=<ACCOUNT-FROM-ABOVE>
+aptos move publish \
+        --named-addresses econia_faucet=$FAUCET_ADDR \
+        --profile econia_faucet_deploy \
+        --assume-yes
+
+# deploy an exchange
+cd ./econia/src/move/econia
+aptos init --profile econia_exchange_deploy # enter "local" for the chain
+export ECONIA_ADDR=<ACCOUNT-FROM-ABOVE>
+aptos move publish \
+        --override-size-check \
+        --included-artifacts none \
+        --named-addresses econia=$ECONIA_ADDR \
+        --profile econia_exchange_deploy \
+        --assume-yes
+
+# run the script
+cargo run -- $APTOS_NODE_URL $APTOS_FAUCET_URL $ECONIA_ADDR $FAUCET_ADDR
```

Comment:
do you have to be in the same directory as the script's `main.rs` for this to work?